src/hooks/usePublicMessages.js

```diff
@@ -48,13 +50,13 @@ export default function usePublicMessages(infoPoint, routes) {
   return useMemo(() => {
     if (!(publicMessages instanceof Array)) return publicMessages;
     return publicMessages.sort(comparePublicMessages).filter((publicMessage) => {
-      if ((routes instanceof Array) && (publicMessage.routeAbbreviation)) {
-        return routes.includes(publicMessage.routeAbbreviation);
+      if (routes instanceof Array && publicMessage.routes) {
+        return (publicMessage.routes.filter((route) => {
+          return routes.includes(route.abbreviation);
+        }).length > 0);
```

Comment:
As is written, I believe you've made is so that you have to include `ALL` as a route in the filter or else general messages will be filtered out.

Not sure if that was intentional - if it was, while it may eventually be a desired feature to be able to filter out general messages, the `ALL` value is just a made up label for them and I don't think we should be relying on it to control that.

This is also why the `|| 'ALL'` condition was applied after all filtering/sorting logic.